### DISCUSSION TITLE
Need help text show like this please im new

### DISCUSSION THREAD

**Discussion by virtualb2b** [fabricjs/fabric.js#9877] [General]
![Text Like This](https://github.com/fabricjs/fabric.js/assets/96039307/1a1266b5-3e0c-4465-a6ae-2e89b63b294c)

im new in fabric js i make double stroke in I-text i need to shapes effect like this any one help me out shear code examples thanks in advance

**Comment by asturur** [comment]
I'm sorry @virtualb2b this opening thread does not let any potential developer understand what you need help with.
Please make an effort to explain questions in details and one per discussion if the overlap is minimal

**Comment by virtualb2b** [comment]
sorry for late reply please check the attachment and giving blow link of
web site i need text effects like this

https://champrosports.com/cslockerroom/index?id=JBS6&LeadTime=&Fabric=ACTIVECLOTH/#/customise/91816576?customDesignId=176102274&basketIndex=3

On Fri, May 10, 2024 at 7:51 PM Andrea Bogazzi ***@***.***>
wrote:

> I'm sorry @virtualb2b <https://github.com/virtualb2b> this opening thread
> does not let any potential developer understand what you need help with.
> Please make an effort to explain questions in details and one per
> discussion if the overlap is minimal
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fabricjs/fabric.js/discussions/9877#discussioncomment-9384613>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AW4XDC5MNNQ7GSXWX6KJDK3ZBTNGZAVCNFSM6AAAAABHPXSLFGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TGOBUGYYTG>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Kind Regards,



Virtual B2B,

We'll Build Your Imagination, Virtual B2B, Is The Icon Of 3D Art.


Sales and Marketing Team,



Contacts:   +92-345-6698544  /  +92-333-0511117
E-Mail:       ***@***.***
Skype:       virtualb2b
URL:          www.virtualb2b.net
FB:            www.facebook.com/onlineweb3d/

**Reply by mhieupham1** [reply]
i think you just need to change the font-family and font-size @virtualb2b
